.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    branches:
+      - master
+      - stable
+name: Build
+jobs:
+  build:
+    runs-on: macos-latest
+    name: Build and deploy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gprbuild
+          path: gprbuild
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-core
+          path: gnatcoll-core
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-bindings
+          path: gnatcoll-bindings
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/langkit
+          path: langkit
+          fetch-depth: 0
+      - run: pushd langkit; git checkout ${{ github.ref }} || true; popd
+      - uses: ada-actions/toolchain@b55aaa7
+        with:
+          distrib: community
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: |
+          set -x -e
+          export ADALIB_DIR=/tmp/ADALIB_DIR
+          export GPR_PROJECT_PATH=$ADALIB_DIR/share/gpr
+          pip install -r langkit/REQUIREMENTS.dev
+          pip install jsonschema
```

Comment:
I think if I understand correctly you're using langkit's req file because LAL's is not adapted because it contains a dep on langkit master, right ? I think instead you should:

1. use libadalang's own `REQUIREMENTS.dev`
2. Edit it to remove the dep on langkit, which is annoying 98% of the time
2. Also add the jsonschema dep so you can remove it from here.

Are you OK to do those changes ?

Reply:
I'm afraid, it's too complicated for me. I just took lines from `utils/travis-install.sh`
```
# Install requirements
pip install -r langkit/REQUIREMENTS.dev
pip install jsonschema
(cd langkit && python setup.py install)

```

Reply:
Ok makes sense, I'll take care of the change in that case :)